DEVELOP.md

```diff
@@ -85,8 +88,71 @@ bundle exec jekyll build --watch
 
 ## Windows Instructions
 
-To be completed...
+Getting Jekyll on Windows is a more lengthy process. First, install [rbenv for Windows](https://github.com/ccmywish/rbenv-for-windows). To do so, open a Powershell terminal, and then run:
+
+```powershell
+$env:RBENV_ROOT = "$HOME\Ruby-on-Windows"
+iwr -useb "https://github.com/ccmywish/rbenv-for-windows/raw/main/tools/install.ps1" | iex
+```
+
+If either command doesn't run correctly, you may lack the ability to run Powershell scripts. To solve this, open an _administrator_ Powershell terminal, and run:
+
+```powershell
+Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser
+```
+
+Then opening another terminal and re-running the installation command for `rbenv` should work. The powershell profile must then be modified to activate rbenv. To do so, run `notepad $PROFILE`, and in the resulting notepad window add in the following lines:
+
+```powershell
+# rbenv for Windows
+$env:RBENV_ROOT = "$HOME\Ruby-on-Windows"
+
+# init rbenv
+& "$env:RBENV_ROOT\rbenv\bin\rbenv.ps1" init
+```
+
+Now open another powershell terminal, and rbenv should automatically start downloading the Ruby toolchain. If it does not, run:
+
+```powershell
+rbenv install 3.3.3
+rbenv global 3.3.3
+```
+
+To check that these commands succeeded, run `rbenv global` and it should output something like `3.3.3-1`.
+
+Note that if it prompts between installing the lite version or the full version, choose the **full version**. Once this is installed, run:
+
+```powershell
+gem install bundler
+```
+
+Then, clone the repository with Git:
+
+```powershell
+git clone https://github.com/NatronGitHub/NatronGitHub.github.io && cd NatronGitHub.github.io
+```
+
+And install the dependencies for Jekyll (which might take quite a while, you can let it run in the background):
+
+```powershell
+bundle install
+```
+
+For reasons we are not entirely aware, this command may fail even when rbenv is installed to a directory that does not require elevated (administrator) permissions. As a temporary fix, open an administrator powershell, and then re-run `bundle install`, which should allow it to work. Finally, to launch the website server, run:
+
+```powershell
+$env:DISABLE_WHITELIST=1 # to load the jekyll plugins we use
+bundle exec jekyll serve
+```
+
+### macOS Instructions
+
+First, you want to get [Homebrew](https://brew.sh/), which is a package manager for macOS that installs development tools and libraries.
+
+Instructions will be fleshed-out in time, but for now, follow the same steps as in the GNU/Linux guide, with the one difference being that you want to use Homebrew to install `rbenv`, by running `brew install rbenv`. The remainder of the steps should be exactly the same as on GNU/Linux.
```

Comment:
See Google's guidelines on [writing timeless documentation](https://developers.google.com/style/timeless-documentation).  Don't make promises for the future, instead word things the way they are currently, that way folks aren't left hoping for a guide that may never arrive!

```suggestion
Follow the same steps as in the GNU/Linux guide, with the one difference being that you want to use Homebrew to install `rbenv`, by running `brew install rbenv`. The remainder of the steps should be exactly the same as on GNU/Linux.
```